.release/ci.hcl

```diff
@@ -0,0 +1,216 @@
+schema = "1"
+
+project "packer" {
+  team = "#team-packer"
+  slack {
+    notification_channel = "C032JM64THT"
+  }
+  github {
+    organization = "hashicorp"
+    repository = "packer"
+    release_branches = [
+        "main",
+        "crt-onboarding"
```

Comment:
Note to remove this branch before merging like the other references in other files